pkg/features/kube_features.go

```diff
@@ -1238,4 +1245,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	HPAScaleToZero: {Default: false, PreRelease: featuregate.Alpha},
 
 	StorageNamespaceIndex: {Default: true, PreRelease: featuregate.Beta},
+
+	SeparateCacheWatchRPC: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
nit: put between `RemainingItemCount` and `ServerSideApply`

Reply:
Done.